src/pybind/mgr/dashboard/module.py

```diff
@@ -517,6 +517,12 @@ class Root(object):
             def default(self, *args, **kwargs):
                 if module.get_module_option('standby_behaviour', 'redirect') == 'redirect':
                     active_uri = module.get_active_uri()
+
+                    if cherrypy.request.path_info == '/api/prometheus_receiver':
+                        module.log.info("Suppressed redirecting alert to active '%s'",
+                                        active_uri)
+                        return None
```

Comment:
Just to confirm, this is a 200? The HTTP 1.1 spec mandates that a 200 on a POST request will be accompanied by a description of the entity created, so we should rather [return a `204` (No Content)](https://datatracker.ietf.org/doc/html/rfc2616#section-9.5) here.